crates/burn-tensor/src/tensor/activation/base.rs

```diff
@@ -78,6 +78,24 @@ pub fn softmax<const D: usize, B: Backend>(tensor: Tensor<B, D>, dim: usize) ->
     tensor.div(tensor_tmp)
 }
 
+/// Applies the softmin function on the input tensor along the given dimension.
+///
+/// `softmin(x_i) = exp(-x_i) / sum_j(exp(-x_j))`
+///
+/// # Notes
+///
+/// The dimension argument `dim` specifies the dimension along which the function will be computed.
+/// It must in the range of `0` and `D-1`.
+pub fn softmin<const D: usize, B: Backend>(tensor: Tensor<B, D>, dim: usize) -> Tensor<B, D> {
+    check!(TensorCheck::dim_ops::<D>("softmax", dim));
+
+    let tensor = tensor.clone() - tensor.detach().min_dim(dim);
```

Comment:
Not sure if we want to subtract `min_dim`. It's for numerical stability, so maybe it should be `max_dim` like softmax ? @laggui

Reply:
Thanks for this @nathanielsimard I wasn't entirely sure what was going on with this line... I did largely copy paste the softmax code for this. I will switch this back to max_dim here.

It appears that max_dim / min_dim has no impact on the test results.